mcs/Makefile

```diff
@@ -6,9 +6,9 @@ SUBDIRS := build jay mcs class nunit24 ilasm tools tests errors docs packages
 
 basic_SUBDIRS := build jay class
 build_SUBDIRS := build class class/aot-compiler tools ilasm
-monodroid_SUBDIRS := build class
+monodroid_SUBDIRS := build class tools
```

Comment:
This is a lot nicer than having the sdk build go to that dir and a  "make PROFILE=xxx"!
I'll apply this to wasm as well.